.github/workflows/ci.yml

```diff
@@ -23,28 +23,33 @@ jobs:
           # cache: "npm" Does't work when project is not in default directory
           # working-directory doesn't work in conjunction with "uses"
       - name: Install backend
-        run: npm install --arch=x64 --platform=linuxmusl
+        run: chmod -R 777 . && docker-compose -f docker-compose-ci.yml run --rm yarn install
         working-directory: ./server
       - name: Test backend
-        run: npm test
+        run: docker-compose -f docker-compose-ci.yml run --rm yarn test
         working-directory: ./server
       - name: Start Docker Backend Environment
-        run: chmod -R 777 . && docker-compose -f docker-compose-ci.yml --env-file ./.env.ci up -d
+        run: docker-compose -f docker-compose-ci.yml --env-file ./.env.ci up -d
         working-directory: ./server
       - name: List all databases (just for debugging)
-        run: sleep 30 && docker exec db psql -U xccup_user -l
+        run: sleep 10 && docker exec db psql -U xccup_user -l
       - name: Setup DB (enable PostGIS and set correct timezone)
-        run: docker exec db psql -U xccup_user xccup_db -c "create extension postgis;" -c "set timezone='Europe/Berlin'"
+        run: docker exec db psql -U xccup_user xccup_db -c "set timezone='Europe/Berlin'"
+        # It looks like that it's no longer necessary to activate the postgis extension
+        # run: docker exec db psql -U xccup_user xccup_db -c "create extension postgis;" -c "set timezone='Europe/Berlin'"
         working-directory: ./server
       - name: Install and build frontend
-        run: yarn install && yarn build-local
+        run: docker-compose run --rm yarn install && docker-compose run --rm yarn build-local
         working-directory: ./client
       - name: Print current server log before testing (just for debugging)
         run: docker logs server
+      - name: Install cypress
+        run: npx cypress install
+        # TODO: Fix cypress caching issue and remove the addional install
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
-          start: yarn serve
+          start: docker-compose run --service-ports --rm client serve
```

Comment:
Wieso denn hier wieder der Client?

Ich dachte dafür ist das Yarn Image, damit das alle "Yarn-Aufgaben" ausführt

Reply:
Der yarn service ist so wie yarn im terminal gedacht.

So wird auch direkt der richtige service verwendet, der auch nachher in der Produktion läuft.